python/ray/dashboard/client/src/pages/dashboard/tune/TuneTable.tsx

```diff
@@ -180,56 +322,125 @@ class TuneTable extends React.Component<
+                  )}
+                  <TableCell className={classes.cell} key="error">
+                    Error
                   </TableCell>
-                  {trial["metrics"] &&
-                    metricNames.map((value) => (
-                      <TableCell className={classes.cell} key={value}>
-                        {trial["metrics"][value]}
-                      </TableCell>
-                    ))}
                 </TableRow>
-              ))}
-          </TableBody>
-        </Table>
+              </TableHead>
+              <TableBody>
+                {trialDetails !== null &&
+                  trialDetails.map((trial, index) => (
+                    <TableRow key={index}>
+                      <TableCell className={classes.cell}>
+                        {trial["trial_id"]}
+                      </TableCell>
+                      <TableCell className={classes.cell}>
+                        {trial["job_id"]}
+                      </TableCell>
+                      <TableCell className={classes.cell}>
+                        {trial["start_time"]}
+                      </TableCell>
+                      {viewableParams.map((value, index) => (
+                        <TableCell className={classes.cell} key={index}>
+                          {trial["params"][value]}
+                        </TableCell>
+                      ))}
+                      <TableCell className={classes.cell}>
+                        {trial["status"]}
+                      </TableCell>
+                      {trial["metrics"] &&
+                        viewableMetrics.map((value, index) => (
+                          <TableCell className={classes.cell} key={index}>
+                            {trial["metrics"][value]}
+                          </TableCell>
+                        ))}
+                      <TableCell className={classes.cell}>
+                        {trial["error"] === "No Error" ? (
+                          "No Error"
+                        ) : (
+                          <Link
+                            component="button"
+                            variant="body2"
+                            onClick={() => {
+                              this.handleOpen(trial["trial_id"]);
+                            }}
+                          >
+                            Show Error
+                          </Link>
+                        )}
+                      </TableCell>
+                    </TableRow>
+                  ))}
+              </TableBody>
+            </Table>
+          </Grid>
+        </Grid>
+        <Dialog fullWidth maxWidth="md" open={open} onClose={this.handleClose}>
+          <DialogContent>
+            {open && (
+              <NumberedLines
+                lines={tuneInfo["trial_records"][errorTrial]["error"]
+                  .trim()
+                  .split("\n")}
+              />
+            )}
+          </DialogContent>
+        </Dialog>
```

Comment:
Consider using the DialogWithTitle component from common/DialogWithTitle.tsx for this. See example usage in pages/dashboard/node-info/NodeInfo.tsx.

Reply:
Done!